src/views/virtualmachines/details/tabs/configuration/details/components/DeletionProtection/DisableDeletionProtectionModal.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { getName, getNamespace } from '@kubevirt-utils/resources/shared';
+import { Button, ButtonVariant, Modal, ModalVariant } from '@patternfly/react-core';
+import { setDeletionProtectionforVM } from '@virtualmachines/details/tabs/configuration/details/components/DeletionProtection/utils/utils';
+
+type DisableDeletionProtectionModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  vm: V1VirtualMachine;
+};
+
+const DisableDeletionProtectionModal: FC<DisableDeletionProtectionModalProps> = ({
+  isOpen,
+  onClose,
+  vm,
+}) => {
+  const { t } = useKubevirtTranslation();
+
+  const submitHandler = () => {
+    setDeletionProtectionforVM(vm, false);
```

Comment:
1. missing camel-case
2. this is a remote call - do we have some error handling here?

Reply:
1. Fixed!
2. We don't. This function is building off of existing functions. @metalice Do you know why we don't have any error handling [here](https://github.com/kubevirt-ui/kubevirt-plugin/blob/0a5a36d38d8035e23f5800f943cadc2804875dec/src/views/virtualmachines/details/tabs/configuration/details/utils/utils.ts#L92)?